certified-connectors/AgilePoint NX/apiProperties.json

```diff
@@ -0,0 +1,58 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Username",
+          "description": "The Username for this api",
+          "tooltip": "Provide the Username",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Password",
+          "description": "The Password for this api",
+          "tooltip": "Provide the Password",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+      "agilepointServerUrl": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "AgilePoint NX Server URL (REST)",
+          "description": "The AgilePoint Server URL",
+          "tooltip": "Provide Your AgilePoint Server URL",
+          "constraints": {
+            "tabIndex": 4,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "AgilePointInstance",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "@connectionParameters('agilepointServerUrl')"

```

Comment:
Is the expectation that the user will provide the entire service URL, or are they only providing a subdomain on the main API domain? If they're not providing a full service URL, then this incidentally won't work and have successful calls, as the main domain is missing.

Reply:
Yes, they have to provide the full-Service URL. That's the expectation from the User.